src/testWithSpringBoot_2_4/java/org/openrewrite/java/spring/boot2/WebSecurityConfigurerAdapterTest.java

```diff
@@ -78,7 +78,7 @@ void someMethod() {}
               public class SecurityConfiguration {
               
                   @Bean
-                  SecurityFilterChain filterChain(HttpSecurity http) throws Exception {
+                  SecurityFilterChain securityConfigurationSecurityFilterChain(HttpSecurity http) throws Exception {
```

Comment:
Is it possible to keep the `filterChain` method name if there is only 1 configuration class? You need the different names if you have multiple, but if there is only one, then using `filterChain` would be nicer I think.